### DISCUSSION TITLE
How to build link between the pages from one Nuxt.js project to another Nuxt.js project within same root folder?

### DISCUSSION THREAD

**Discussion by Aravinda93** [nuxt/nuxt#22871] [Questions]
I have a Nuxt.js 3 0r Vue 3 based application that contains a structure similar to this:

```
📦root
 ┣ 📂commons
 ┗ 📂project1
      ┣ 📜pages
           ┗ 📜landing.vue
 ┗ 📂project2
       ┣ 📜pages
            ┗ 📜myHome.vue
```

I would like to access the `myHome.vue` in `project2` from `project1` but am unable to do so using the `NuxtLink`:

```
<NuxtLink to="/myHome"> Home </NuxtLink> 
```

I also tried by providing the folder name:
```
<NuxtLink to="project2/myHome"> Home </NuxtLink> 
```

Still not working. I have the re-usable components in my `common` folder and `project1` and `project2` extend the `common` in nuxt.config: `extends: '../common',`.

Can someone please let me know how can I build the link between the pages in `project1` and `project2` as per the standard approach?

**Comment by xjccc** [comment]
https://nuxt.com/docs/guide/directory-structure/components#custom-directories

i dont know this if can help you ?
